### ISSUE TITLE
Still having problems with sequences that are not indented

### ISSUE THREAD

**Issue by ziadsawalha** [jeremyfa/yaml.js#7]
This fails in yaml.js, but is valid YAML.

``` yaml:
- sequence1:
  - long_string_in_unindented_sequence: |
         line1
         line2
  - name: something
```

it should yield this:

``` json:
[
  {
    "sequence1": [
      {
        "long_string_in_unindented_sequence": "line1\nline2\n"
      }, 
      {
        "name": "something"
      }
    ]
  }
]
```

**Comment by jeremyfa** [comment]
This will be fixed in the rewrite I am working on currently.

**Comment by jeremyfa** [comment]
Fixed in `v0.2.0`

**Comment by ziadsawalha** [comment]
Thanks!
